### PR TITLE
First swing at Kerbside docs.

### PR DESCRIPTION
**Pull request by mikalstill**

### CODE REVIEW

File: docs/components/kerbside.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
+# Kerbside
+
+Kerbside is a [SPICE](https://spice-space.org/) protocol native VDI proxy
+responsible for providing rich VDI experiences to users of Shaken Fist, oVirt,
+OpenStack, or any other cloud where the hypervisor is capable of providing SPICE
+consoles over the network.
+
+The SPICE protocol has existed for a long time ago, and still represents the
```
Comment (fifieldt) [inline docs/components/kerbside.md]: for a long time*

File: docs/components/kerbside.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
+# Kerbside
+
+Kerbside is a [SPICE](https://spice-space.org/) protocol native VDI proxy
+responsible for providing rich VDI experiences to users of Shaken Fist, oVirt,
```
Comment (fifieldt) [inline docs/components/kerbside.md:4]: s/responsible for providing/that provides

File: docs/components/kerbside.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+OpenStack, or any other cloud where the hypervisor is capable of providing SPICE
+consoles over the network.
+
+The SPICE protocol has existed for a long time ago, and still represents the
+richest and most performant option for remote desktops using Open Source
+technologies. However before Kerbside, SPICE adoption was limited because the
+console was generally provided in the form of a HTML5 transcoded interface in
+a web client. Unfortunately these HTML5 interfaces do not support many of the
+more novel features of the SPICE protocol, nor do they support high resolution
+desktops. By avoiding transcoding to a HTML5 client, we avoid these problems.
```
Comment (fifieldt) [inline docs/components/kerbside.md]: Alternative lower verbosity version:

Before Kerbside, consoles were generally provided by a HTML5 transcoded interface in a web client. Unfortunately, HTML5 interfaces do not support many of the more novel features of the SPICE protocol, nor do they support high resolution desktops. Kerbside avoids these problems, and brings SPICE to the masses.

File: docs/components/kerbside.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+## Kerbside is a proxy, not a complete user interface
+
+Whilst Kerbside presents a simple administrative interface over HTTP and has
+REST APIs for orchestrating consoles, it is not intended as a complete SPICE
+desktop VDI solution. It is intended that Kerbside itself is orchestrated by
+an external system. That is, in order for a desktop to be presented to a user
+the following steps need to occur:
+
+* The user requests a desktop via an external user interface that we call "the
```
Comment (fifieldt) [inline docs/components/kerbside.md:34]: Since these are a series of steps that must occur in order, one could consider an ordered list.

**Review by fifieldt — APPROVED** [review]
This is most excellent.

I've added a few in-line comments which you may choose incorporate at your option.

**Review by fifieldt — APPROVED** [review]
Lgtm

**Review by fifieldt — APPROVED** [review]

**Comment by fifieldt** [comment]
Needs pymdownx.superfences

**Review by fifieldt — APPROVED** [review]

**Comment by mikalstill** [comment]
Yeah, the weird bit is I don't know why it needs it...

**Pull request merged**